components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/step/impl/DefaultStepHandler.java

```diff
@@ -742,9 +748,17 @@ protected void doAuthentication(HttpServletRequest request, HttpServletResponse
                 return;
             }
 
+            if (LOG.isDebugEnabled() && context.getSubject() != null) {
+                LOG.debug("GIT-22890:User : " + context.getSubject().getUserName()
+                        + " authenticated with the authenticator: " + authenticator.getName());
+            }
             // Set authorized organization and user resident organization for B2B user logins.
             if (context.getSubject() != null && isLoggedInWithOrganizationLogin(authenticatorConfig)) {
                 String userResidentOrganization = resolveUserResidentOrganization(context.getSubject());
+                if (LOG.isDebugEnabled()) {
+                    LOG.debug("GIT-22890:User : " + context.getSubject().getUserName()
```

Comment:
lets log fullQualifiedUsername

Reply:
Addressed in [d6547fc](https://github.com/wso2/carbon-identity-framework/pull/5616/commits/d6547fceae241d6de5a740e7b1b8dd24c563c98f)